corehq/motech/repeaters/repeater_generators.py

```diff
@@ -503,26 +503,33 @@ def get_case_block(self, registry_helper, repeat_record, couch_user):
                 "date_opened": self.intent_case.opened_on
             }
 
+        updates = self.get_case_updates(self.intent_case, self.includes, self.excludes)
+        if self.copy_case_id:
+            copy_from = self._get_registry_case(
+                self.copy_domain, self.copy_case_id, self.copy_case_type, False,
+                registry_helper, repeat_record, couch_user
+            )
+            updates.update(self.get_case_updates(copy_from, self.copy_includelist, self.copy_excludelist))
```

Comment:
Is this `update` precedence correct?  I'd imagine if any properties are set on both the intent case and the `copy_case`, we'd want the intent case to win out, as it exists just for this purpose.

Reply:
good point, I hadn't thought of that: aa05b62362f